src/components/atoms/bar-chart-content.tsx

```diff
@@ -102,6 +105,24 @@ export const BarChartContent: FC<BarChartContentProps> = ({
     <TrendLine lineWidth={3} color={primaryColor} {...props} />
   );
 
+  const Label: React.ReactNode = (props: BarChildProps) => {
+
+    const {x, y, bandwidth, data} = props;
+    return data.map((value: {value: number}, index: number) =>
+      index === data.length - 1 ? (
+        <View
+          accessible={true}
+          style={[styles.label, {top: y(value.value) - 35, right:(bandwidth /2) -  13 }]} key={`label-${value.value}`}>
```

Comment:
Needs linting, `npm run lint:fix` from the command line if you don't have eslint plugin set up in your editor

Reply:
Don't worry about the console output, just commit the changes it makes automatically